delft3dworker/test_models.py

```diff
@@ -339,10 +342,11 @@ def test_export_export(self):
         self.assertEqual(len(zf.namelist()), 1)
 
     @patch('delft3dworker.tasks.chainedtask.delay', autospec=True)
-    def test_start_scene(self, mocked_task_delay):
-        # mocked_task_delay.return_value = {"info": {}, 'id': 22, 'state': ""}
-        mocked_task_delay.return_value.task_id = 22
-        mocked_task_delay.return_value.state = "PROCESSING"
+    def test_start_scene(self, mockchainedtask):
```

Comment:
Fedor heeft in mijn code reviews de suggestie gedaan om een mocked Function volgens PEP8 Function naming standards te namen: https://www.python.org/dev/peps/pep-0008/#function-names
"Function names should be lowercase, with words separated by underscores as necessary to improve readability."